client/tests/integration/upgrade.rs

```diff
@@ -197,7 +199,7 @@ fn executor_upgrade_should_revoke_removed_permissions() -> Result<()> {
 
 #[test]
 fn migration_fail_should_not_cause_any_effects() {
-    let (_rt, _peer, client) = <PeerBuilder>::new().with_port(10_995).start_with_runtime();
+    let (_rt, _peer, client) = <PeerBuilder>::new().with_port(10_999).start_with_runtime();
```

Comment:
Wouldn't it conflict with `connected_peers_with_f_1_0_1` which used port 11_000?

Reply:
It might... changed to 10_998